tests/golden/check-footnotes/check-footnotes.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "We report broken links inside footnotes" {
+  xrefcheck \
+  | prepare > /tmp/check-ignored.test || true
+
+  diff /tmp/check-ignored.test expected.gold \
+    --ignore-space-change \
+    --ignore-blank-lines \
+    --new-file # treat absent files as empty
+
+  rm /tmp/check-ignored.test
+}
+
+@test "We're not threating footnotes as link references" {
```

Comment:
Fixed a typo and added a bit of context information for future readers/maintainers:

```suggestion
@test "We're not treating footnotes as 'shortcut reference links'" {
  # See: https://github.com/serokell/xrefcheck/issues/155
```

Reply:
Forgot to add the link to #155

Reply:
Oh, thanks (fixed)